secboot/secboot_sb_test.go

```diff
@@ -121,7 +129,7 @@ func (s *secbootSuite) TestCheckTPMKeySealingSupported(c *C) {
 		restoreEfiVars := efi.MockVars(vars, nil)
 		defer restoreEfiVars()
 
-		err := secboot.CheckTPMKeySealingSupported()
+		err := secboot.CheckTPMKeySealingSupported(secboot.TPMProvisionFull)
```

Comment:
do we need a test where mode is not full?

Reply:
Excellent point, I improved the coverage here now as well